app/javascript/proconf-landing--draft/src/routes/public/routes.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Route } from "react-router-dom";
+
+import { Home } from "./routes/Home";
+import { Page } from "../../components";
+import { EpisodeDetails } from "./routes/EpisodeDetails";
+import { About } from "./routes/About";
+import { Donate } from "./routes/Donate";
+import { Switch } from "react-router";
+import { NotFound } from "./routes/NotFound";
+
+export default () => {
+  return (
+    <Page>
+      <Page.Header>
+        <Page.Header.Logo />
+        <Page.Header.Navbar />
+        <Page.Header.Social />
+      </Page.Header>
+      <Switch>
+        <Route path='/episodes/:id' component={EpisodeDetails} />
+        <Route path='/episodes/:id/list/:type' component={EpisodeDetails} />
```

Comment:
Should you move `<Route path='/episodes/:id/list/:type' component={EpisodeDetails} />` before `<Route path='/episodes/:id' component={EpisodeDetails} />`?
Is it works correct?

Reply:
I delete this useless route